app/templates/components/container/datafruits-player.hbs

```diff
@@ -16,7 +16,7 @@
       </a>
     {{else if playing}}
       <a class="jp-pause" tabindex="1" {{action "pause"}}>
-        {{t "player.pause"}}
+        {{t (if this.isLive "player.stop" "player.pause")}}
```

Comment:
So I think the `isLive` computed property is a bit of hack...it really only looks at the title to see if it starts with "LIVE". Its totally possible to change the title to something else, even if its still playing the live stream. 

I think in this case we can use the `playingPodcast` property to achieve this.

Reply:
Addressed in 
7839b97